AutomatedTesting/Gem/PythonTests/scripting/scripting_utils/scripting_tools.py

```diff
@@ -0,0 +1,98 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+from editor_python_test_tools.utils import TestHelper as helper
+from PySide2 import QtWidgets, QtTest, QtCore
+import editor_python_test_tools.pyside_utils as pyside_utils
+import azlmbr.editor as editor
+import azlmbr.bus as bus
+from scripting_utils.scripting_constants import (SCRIPT_CANVAS_UI, ASSET_EDITOR_UI, NODE_PALETTE_UI, NODE_PALETTE_QT,
+                                                 TREE_VIEW_QT, SEARCH_FRAME_QT, SEARCH_FILTER_QT, SAVE_STRING,
+                                                 SAVE_ASSET_AS, WAIT_TIME_3)
+
+
+def save_script_event_file(self, file_path):
+    """
+    function for saving a script event file with a user defined file path. Requires asset editor qt object to be initialized
+    and any required fields in the asset editor to be filled in before asset can be saved.
+
+    param self: the script object calling this function
+    param file_path: full path to the file as a string
+    returns: none
+    """
+    editor.AssetEditorWidgetRequestsBus(bus.Broadcast, SAVE_ASSET_AS, file_path)
+    action = pyside_utils.find_child_by_pattern(self.asset_editor_menu_bar, {"type": QtWidgets.QAction, "iconText": SAVE_STRING})
+    action.trigger()
+    # wait till file is saved, to validate that check the text of QLabel at the bottom of the AssetEditor,
+    # if there are no unsaved changes we will not have any * in the text
+    label = self.asset_editor.findChild(QtWidgets.QLabel, "textEdit")
+    return helper.wait_for_condition(lambda: "*" not in label.text(), WAIT_TIME_3)
+
+
+def initialize_asset_editor_qt_objects(self):
+    """
+    function for initializing qt objects needed for testing around asset editor
+
+    param self: teh script object calling this function.
```

Comment:
NIT (typo): teh

Reply:
Additionally, the rest of the tests that return none are logged in the docstrings. Let's keep things consistent and either have all that return none be explicit or all the tests that return none be implicit in their docstrings. Dealer's choice here on direction :)

Reply:
It might be clearer if we are explicit with the return type even if there isn't a return statement. Added a simple 'returns none' to these function comments